tests/integration/backward_compatible/proxy/cp/count_down_latch_test.py

```diff
@@ -44,14 +45,23 @@ def test_await_latch_zero_timeout(self):
         self.assertFalse(latch.await_latch(0))
 
     def test_await_latch_with_timeout(self):
-        timeout = 0.1
+        timeout = 1
         latch = self.get_latch(1)
         start = get_current_timestamp()
         self.assertFalse(latch.await_latch(timeout))
         time_passed = get_current_timestamp() - start
+
+        expected_time_passed = timeout
+        if os.name == "nt":
+            # On Windows, we were getting random test failures due to expected
+            # time passed being slightly less than the timeout. This is due to
+            # the low time resolution there (15-16ms). If we are on Windows, we
+            # lower our expectations and settle for a slightly lower value.
+            expected_time_passed *= 0.95
+
         self.assertTrue(
-            time_passed >= timeout,
-            "Time passed is less than %s, which is %s" % (timeout, time_passed),
+            time_passed >= expected_time_passed,
+            "Time passed is less than %s, which is %s" % (timeout, expected_time_passed),
```

Comment:
I think you renamed the wrong variable in this line

Reply:
Fixed in https://github.com/hazelcast/hazelcast-python-client/pull/469/commits/9f52471f47439ccdbe7cb30fc265b3c611e82cdf